ci/go-publisher-ossf-slsa3.yml

```diff
@@ -0,0 +1,22 @@
+name: SLSA Go releaser
```

Comment:
Please add a header disclaimer that this workflow uses third party actions. See the [guidance](https://github.com/actions/starter-workflows/blob/main/.github/pull_request_template.md).
```
# This workflow uses actions that are not certified by GitHub.
# They are provided by a third-party and are governed by
# separate terms of service, privacy policy, and support
# documentation.
```

Reply:
Done

Reply:
Third party is in sense that it is not directly owned by GitHub.

Reply:
Unresolving the comment as I do not see this updated in file.

Reply:
my bad, updated now. Note that I don't fully understand the motivation behind this disclaimer. An Action can be replaced by an internal library or piece of code and achieve the same functionality, without the need for the disclaimer.

I would understand it better if it was required for 3P actions external to this repo .